sdk/containerservice/arm-containerservice/src/models/mappers.ts

```diff
@@ -1670,13 +1677,28 @@ export const ManagedClusterOidcIssuerProfile: coreClient.CompositeMapper = {
   },
 };
 
+export const ManagedClusterNodeResourceGroupProfile: coreClient.CompositeMapper =
+  {
+    type: {
+      name: "Composite",
+      className: "ManagedClusterNodeResourceGroupProfile",
+      modelProperties: {
+        restrictionLevel: {
+          serializedName: "restrictionLevel",
+          type: {
+            name: "String",
+          },
+        },
+      },
+    },
+  };
+
 export const ContainerServiceNetworkProfile: coreClient.CompositeMapper = {
   type: {
     name: "Composite",
     className: "ContainerServiceNetworkProfile",
     modelProperties: {
       networkPlugin: {
-        defaultValue: "kubenet",
```

Comment:
interesting the default value is removed. @kazrael2119 could you find relevant swagger and typescript definition for me?

Reply:
swagger: https://github.com/kazrael2119/azure-rest-api-specs/blob/86d9b39c82444344723eb239f2f16aa47280dad7/specification/containerservice/resource-manager/Microsoft.ContainerService/aks/stable/2024-09-01/managedClusters.json#L3844-L3845

sdk code definition: https://github.com/kazrael2119/azure-sdk-for-js/blob/e604dfdbcd2604f655ec1b32900153bb58990e19/sdk/containerservice/arm-containerservice/src/models/index.ts#L2577-L2596